Exercise-15/js/index.js

```diff
@@ -0,0 +1,84 @@
+import videoData from "../assets/video.json" assert { type: "json" };
+import posterData from "../assets/posters.json" assert { type: "json" };
+
+//-----videos section-----
+const videoContainer = document.getElementById("videoContainer");
+const videoTitle = document.getElementById("videoTitle");
+const videoDescription = document.getElementById("videoDescription");
+
+function addVideoDetails() {
+  //adding video
+  const video = document.createElement("video");
+  video.src = videoData.videoUrl;
+  video.setAttribute("width", "100%");
+  video.controls = true;
+  videoContainer.appendChild(video);
+
+  //adding video title
+  const title = document.createElement("span");
+  title.innerText = videoData.title;
+  videoTitle.appendChild(title);
```

Comment:
could let me know if you could construct the video title & video description and directly append to videoContainer itself instead of creating two separate divs with id's and constructing another two elements as child to these divs

Reply:
I have removed the divs and appended the title and description directly to videoContainer